main.py

```diff
@@ -4,7 +4,79 @@
 
 
 def main() -> None:
-    pass
+    elf_race = Race.objects.create(name="elf", description="The magic race")
```

Comment:
you should get all data for objects from `players.json`!! don't use hardcode data